index.html

```diff
@@ -1,2 +1,12 @@
 <html>
+    <head>
+        <style>
+            h2{
+                color:'red'
```

Comment:
Please end your css lines with a semi colon even if it is the last line because beginners are watching!

Reply:
Sure thing